app/config/AppConfig.scala

```diff
@@ -64,4 +66,21 @@ class AppConfig @Inject()(servicesConfig: ServicesConfig, configuration: Configu
   def allowListEnabled: Boolean = configuration.get[Boolean]("features.allowListEnabled")
 
   def internalAuthToken: String = configuration.get[String]("internal-auth.token")
+
+  def addressLookupFrontendUrl: String = {
+    if (isEnabled(StubAddressLookupJourney)) {
+      servicesConfig.baseUrl("emcs-tfe-chris-stub")
```

Comment:
This line is untested - it might be worth adding in tests for AppConfig's URLs based on the various feature switches. Then at least we know that "if X is on it looks for Y URL, otherwise it looks for Z URL"